### PR TITLE
Changes required for Polyester and ReactNativeHost

### PR DESCRIPTION
**Pull request by BarinderGrewal**
#### Please select one of the following
- [ ] I am removing an existing difference between facebook/react-native and microsoft/react-native :thumbsup:
- [ ] I am cherry-picking a change from Facebook's react-native into microsoft/react-native :thumbsup:
- [ ] I am making a fix / change for the macOS implementation of react-native
- [x] I am making a change required for Microsoft usage of react-native

Changes required for Polyester and ReactNativeHost

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native/pull/177)

### CODE REVIEW

File: ReactAndroid/src/main/java/com/facebook/react/views/text/ReactFontManager.java [hunk 1]
```diff
@@ -62,6 +65,21 @@ public static ReactFontManager getInstance() {
     return getTypeface(fontFamilyName, style, 0, assetManager);
   }
 
+  public @Nullable Typeface getTypeface(
```
Comment (acoates-ms) [inline ReactAndroid/src/main/java/com/facebook/react/views/text/ReactFontManager.java]: Can we instead extend the existing addCustomFont method, to have another override which takes a Typeface directly?  That way we are not introducing another font loading path. -- it'll be a smaller diff from the OSS version, and we can look at upstreaming that change to OSS?

Comment (BarinderGrewal) [reply ReactAndroid/src/main/java/com/facebook/react/views/text/ReactFontManager.java]: Sure. I will have a look at this.

Comment (BarinderGrewal) [reply ReactAndroid/src/main/java/com/facebook/react/views/text/ReactFontManager.java]: This is what I understood from the flow.
If I add an overridden addCustomFont method which accepts Typeface, this typeface should also be created for which we require our createTypeface method. And in addCustomFont we will have to add this typeface to map(cache) for faster access later. Whenever any component needs to apply this typeface, they will have to use getTypeface method which will return the typeface from the map. This flow will not remove the methods we added but instead add one more method to this class.
Please correct me if wrong as I may not have understood your point properly.

**Comment by acoates-ms** [comment]
Obviously stale...

**Pull request closed without merging**